requirements/requirements.in

```diff
@@ -1,5 +1,9 @@
+
 coverage
 django
 pytest
 pytest-django
 pytest-randomly
+furo
+Sphinx
+sphinxcontrib-spelling
```

Comment:
no docs requirements in test environments, just slows down all tests unnecessarily

docs requirements will go in `docs/requirements.txt`